contributor-growth/drafts/recruiting-playbook.md

```diff
@@ -0,0 +1,133 @@
+# Recruiting Contributors
+*Status: Draft; Missing Components*
+/help-wanted
+
+## Summary 
+
+Recruiting and retention go hand in hand. If you are thinking "how can we add 
+more amazing contributors?" and "where do we find them?" the projects leadership
+ needs to think about how you keep folks around, too. If your goal is to limit 
+ to only single contributions aka drive-bys then this might not apply to you. 
+ But once you open the gates to more contributors of any type, balancing 
+ maintainers time becomes the next challenge. This is why it's important to 
+ think of recruiting and retention initiatives together rather than separate. 
+
+For the purpose of this document, we will dig into the how and where to do 
+outreach [TODO other retention related docs and info; contributor ladder]
+
+Outreach depends on your audience, how they consume information, and their 
+motivations to help an open source project.  
+
+## Where
+Always be recruiting. Think of ways to always promote your needs and areas of 
+growth for the project anytime you have a chance. Be as specific as you can when
+ you are communicating though. The more ambiguous it sounds, the less likely 
+ someone is to check it out or take interest. Talk about roles (example: 
+ reviewers in a certain area/subject, community managers, project managers, go 
+ developers) and provide any kind of guides, role descriptions, contributor 
+ ladders as you have. The more of these documents that you have available in 
+ your project repository the easier it will be for you in explaining how your 
+ project works to potential new contributors. 
+
+General areas with specifics below:
+- your repo via labels, issues calling out help wanted areas, project boards, 
+and other organization
+- a contributor site/area of the project site for contributors
+- mailing list adverts and promotions (weekly help wanted?)
+- in-person or virtual talks and workshops about the project
+- section during community meeting 
+- project twitter announcements 
+
+## Specific Projects & Programs 
+
+### Your README and project website  
+What: Adding the links to how to contribute and what you are contributing to on 
+your main README is the most effective way 
+Links to good examples: TODO include containerd 'now recruiting'
+
+### Contributor Documentation
+Contributor Guide (for all contributors)
```

Comment:
These need extra new lines after each sentence or use bullets to prevent it from rendering as a single runon sentence in markdown.